tests/pausing_test.go

```diff
@@ -361,8 +360,7 @@ var _ = Describe("[rfe_id:3064][crit:medium][vendor:cnv-qe@redhat.com][level:com
 
 				By("Trying to vnc into the VM")
 				_, err = virtClient.VirtualMachineInstance(vm.ObjectMeta.Namespace).VNC(vm.ObjectMeta.Name)
-				Expect(err).To(HaveOccurred())
-				Expect(err.Error()).To(ContainSubstring("VMI is paused"))
+				Expect(err).ToNot(HaveOccurred())
```

Comment:
@andreyod Can you please add an expecter to test the console output of `PrintWarningForPausedVMI`?

Reply:
@andreyod Actually I am not sure if it's possible to catch the warning output on the console stream IO. I think maybe it's worth to create a K8S event of when VMI is paused.

Reply:
Yes. We can't catch the warning here as it is in virtctl and not on console stream IO

Reply:
@enp0s3  So you propose to create K8S event when client is trying to open console to paused VM?

Reply:
@andreyod Let's first check whether the UI is capable of checking the VMI status condition. If so, the event won't be needed since the front-end can react to the condition. Otherwise let's propose to add an event

Reply:
Yes. UI is capable of checking the VMI status condition so don't need an event.